frontend/app/components/changelog.js

```diff
@@ -0,0 +1,5 @@
+import Component from '@ember/component';
```

Comment:
```suggestion
import Component from '@glimmer/component';
```
ember octane prefers glimmer :)